precompiles/xvm/src/mock.rs

```diff
@@ -244,6 +244,7 @@ impl XvmCall<AccountId> for MockXvmWithArgsCheck {
         _source: AccountId,
         target: Vec<u8>,
         input: Vec<u8>,
+        _value: Balance,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `_value` field in the function signature is not used. Consider removing it to avoid confusion.
```diff
-        _value: Balance,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->